src/commands/exportApi.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { getResourceGroupFromId } from "@microsoft/vscode-azext-azureutils";
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import * as fs from "fs-extra";
+import * as path from "path";
+import * as vscode from "vscode";
+import { ApiCenterService } from "../azure/ApiCenter/ApiCenterService";
+import { DefinitionFormat, openapi } from "../constants";
+import { ext } from "../extensionVariables";
+import { ApiVersionDefinitionTreeItem } from "../tree/ApiVersionDefinitionTreeItem";
+import { createTemporaryFolder } from "../utils/fsUtil";
+export namespace ExportAPI {
+    export async function exportApi(
+        context: IActionContext,
+        node?: ApiVersionDefinitionTreeItem): Promise<void> {
+        if (!node) {
+            node = await ext.treeDataProvider.showTreeItemPicker<ApiVersionDefinitionTreeItem>(`${ApiVersionDefinitionTreeItem.contextValue}-${openapi}`, context);
```

Comment:
We need to be able to select any definition, not only openai.
(Note: the showTreeItemPicker support searching by Regex)

Reply:
thanks! updated.